aya-obj/src/obj.rs

```diff
@@ -231,6 +231,7 @@ pub enum ProgramSection {
     },
     Lsm {
         name: String,
+        sleepable_supported: bool,
```

Comment:
I think this should be just `sleepable`?

Reply:
I totally agree. I was trying to keep it consistent with XDP which uses `frags_supported`.

Maybe change both?